.github/workflows/soundness.yml

```diff
@@ -51,6 +51,10 @@ on:
         type: string
         description: "Container image for the shell check job. Defaults to latest Swift Ubuntu image."
         default: "swift:6.0-noble"
+      local_swift_dependencies_check_enabled:
+        type: boolean
+        description: "Boolean to enable the local swift dependencies check job. Defaults to true."
+        default: true
```

Comment:
All the Swift packages that are built into the toolchain will have local Swift dependencies, eg. https://github.com/swiftlang/swift-format/blob/d143a8a59c89aba5bdadf96787f7f41197ce6b71/Package.swift#L173-L175. I’m thus not sure if we want to enable this by default.

Reply:
I agree, this should be disabled by default. Users should be able to opt-in for this check. Apart from that I think this is very useful in some cases. :)

Reply:
Changed the the default value to **false**.